Documentation/redirects.txt

```diff
@@ -23,3 +23,5 @@
 "cmdref/cilium-dbg_statedb_node-addresses.md" "cmdref/cilium-dbg_statedb.md"
 "cmdref/cilium-dbg_statedb_routes.md" "cmdref/cilium-dbg_statedb.md"
 "cmdref/cilium-dbg_statedb_sysctl.md" "cmdref/cilium-dbg_statedb.md"
+"security/cassandra.rst" "security/network/proxy/envoy.rst"
```

Comment:
I'd prefer to link these directly to the "Go extensions" section but I tried to link to an anchor and the CI failed silently, so I'm guessing we just don't have that capability currently. I guess this will do.